natgeo/mortar
# [SYTO-1141] Checking change that wasn't pushed with last PR

Adds right padding to to headline.

To test:

- Pull SYTO-1141-2
- gulp serve
- navigate to http://localhost:3000/components/#video-playlist
- reduce viewport to 417-419px, verify heading has proper spacing in relation to right of the viewport
